gaphor/diagram/copypaste.py

```diff
@@ -3,8 +3,9 @@
 The `copy()` function will return all values serialized, either as string
 values or reference id's.
 
-The `paste()` function will resolve those values. Based on the elements
+The `paste_link()` function will resolve those values. Based on the elements
 that are already in the model,
+`paste_full()` will try copy every element involved.
```

Comment:
How about:

The `paste_link()` function will resolve those values and place instances of them on the diagram using the same defining element. The `paste_full()` function is similar, but it creates new defining elements for the elements being pasted.

Reply:
I improved the docstring with your suggestion :+1: